find . -type f -exec sed -i 's*#!/usr/local/bin/perl*#!/usr/bin/perl*' {} \;

diff -Nru Tk-804.027.vanilla/Compound/compound_demo Tk-804.027/Compound/compound_demo
--- Tk-804.027.vanilla/Compound/compound_demo	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Compound/compound_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Compound;
 my $mw = MainWindow->new;
diff -Nru Tk-804.027.vanilla/Contrib/TickerTape Tk-804.027/Contrib/TickerTape
--- Tk-804.027.vanilla/Contrib/TickerTape	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/TickerTape	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 =head1 NAME
 
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-15.5.pl Tk-804.027/Contrib/book-examples/f-15.5.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-15.5.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-15.5.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 15.5, p153
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.1.pl Tk-804.027/Contrib/book-examples/f-16.1.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.1.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.1.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.1, p159
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.10.pl Tk-804.027/Contrib/book-examples/f-16.10.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.10.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.10.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.2.pl Tk-804.027/Contrib/book-examples/f-16.2.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.2.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.2.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.2, p161
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.3.pl Tk-804.027/Contrib/book-examples/f-16.3.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.3.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.3.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.3, p162
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.6.pl Tk-804.027/Contrib/book-examples/f-16.6.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.6.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.6.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.6, p165
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.7.pl Tk-804.027/Contrib/book-examples/f-16.7.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.7.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.7.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.7, p166
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.8.pl Tk-804.027/Contrib/book-examples/f-16.8.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.8.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.8.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/f-16.9.pl Tk-804.027/Contrib/book-examples/f-16.9.pl
--- Tk-804.027.vanilla/Contrib/book-examples/f-16.9.pl	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/f-16.9.pl	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
diff -Nru Tk-804.027.vanilla/Contrib/book-examples/lusol-f16.3 Tk-804.027/Contrib/book-examples/lusol-f16.3
--- Tk-804.027.vanilla/Contrib/book-examples/lusol-f16.3	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/book-examples/lusol-f16.3	2005-03-11 20:36:24.000000000 +0100
@@ -16,7 +16,7 @@
 first non-strict/with symbolic references, the second strict/hard refs.
 
 -------------------------------- 1 -------------------------------------------
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.3, p162
@@ -43,7 +43,7 @@
 
 MainLoop;
 -------------------------------- 2 -------------------------------------------
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.3, p162
diff -Nru Tk-804.027.vanilla/Contrib/fd_demo Tk-804.027/Contrib/fd_demo
--- Tk-804.027.vanilla/Contrib/fd_demo	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/fd_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::FileDialog;
diff -Nru Tk-804.027.vanilla/Contrib/testlistbox Tk-804.027/Contrib/testlistbox
--- Tk-804.027.vanilla/Contrib/testlistbox	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/Contrib/testlistbox	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 use Tk;
 use Tk::TiedListbox;
 
diff -Nru Tk-804.027.vanilla/DragDrop/drag_test Tk-804.027/DragDrop/drag_test
--- Tk-804.027.vanilla/DragDrop/drag_test	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/DragDrop/drag_test	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop qw(Sun);
diff -Nru Tk-804.027.vanilla/DragDrop/has_sun Tk-804.027/DragDrop/has_sun
--- Tk-804.027.vanilla/DragDrop/has_sun	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/DragDrop/has_sun	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
diff -Nru Tk-804.027.vanilla/DragDrop/local_test Tk-804.027/DragDrop/local_test
--- Tk-804.027.vanilla/DragDrop/local_test	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/DragDrop/local_test	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop;
diff -Nru Tk-804.027.vanilla/DragDrop/motion_test Tk-804.027/DragDrop/motion_test
--- Tk-804.027.vanilla/DragDrop/motion_test	2003-07-19 10:38:57.000000000 +0200
+++ Tk-804.027/DragDrop/motion_test	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop;
diff -Nru Tk-804.027.vanilla/DragDrop/selection Tk-804.027/DragDrop/selection
--- Tk-804.027.vanilla/DragDrop/selection	2003-08-10 12:47:51.000000000 +0200
+++ Tk-804.027/DragDrop/selection	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
diff -Nru Tk-804.027.vanilla/DragDrop/site_test Tk-804.027/DragDrop/site_test
--- Tk-804.027.vanilla/DragDrop/site_test	2004-04-03 19:57:54.000000000 +0200
+++ Tk-804.027/DragDrop/site_test	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DropSite;
diff -Nru Tk-804.027.vanilla/Event/event_demo Tk-804.027/Event/event_demo
--- Tk-804.027.vanilla/Event/event_demo	2003-07-19 10:38:59.000000000 +0200
+++ Tk-804.027/Event/event_demo	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk::Event;
 Tk::Event::HandleSignals;
diff -Nru Tk-804.027.vanilla/Event/findEvent Tk-804.027/Event/findEvent
--- Tk-804.027.vanilla/Event/findEvent	2003-07-19 10:38:59.000000000 +0200
+++ Tk-804.027/Event/findEvent	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 my @names = qw(Tcl_CreateEventSource Tcl_DeleteEventSource Tcl_DeleteEvents
diff -Nru Tk-804.027.vanilla/Extensions/ImageBack/background Tk-804.027/Extensions/ImageBack/background
--- Tk-804.027.vanilla/Extensions/ImageBack/background	2003-07-19 10:38:59.000000000 +0200
+++ Tk-804.027/Extensions/ImageBack/background	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::ImageBack;
 use Tk::Photo;
diff -Nru Tk-804.027.vanilla/HList/Tix2perl Tk-804.027/HList/Tix2perl
--- Tk-804.027.vanilla/HList/Tix2perl	2003-07-19 10:38:59.000000000 +0200
+++ Tk-804.027/HList/Tix2perl	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk::Pretty;
 
diff -Nru Tk-804.027.vanilla/JPEG/t/Read.t Tk-804.027/JPEG/t/Read.t
--- Tk-804.027.vanilla/JPEG/t/Read.t	2004-03-27 11:28:26.000000000 +0100
+++ Tk-804.027/JPEG/t/Read.t	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 print "1..1\n";
 use Tk;
 require Tk::JPEG;
diff -Nru Tk-804.027.vanilla/JPEG/t/more.t Tk-804.027/JPEG/t/more.t
--- Tk-804.027.vanilla/JPEG/t/more.t	2004-03-27 11:29:00.000000000 +0100
+++ Tk-804.027/JPEG/t/more.t	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Test;
 use Tk;
diff -Nru Tk-804.027.vanilla/JPEG/tkjpeg Tk-804.027/JPEG/tkjpeg
--- Tk-804.027.vanilla/JPEG/tkjpeg	2004-02-28 18:10:55.000000000 +0100
+++ Tk-804.027/JPEG/tkjpeg	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::JPEG;
diff -Nru Tk-804.027.vanilla/TextList/textlist_demo Tk-804.027/TextList/textlist_demo
--- Tk-804.027.vanilla/TextList/textlist_demo	2003-07-20 13:30:23.000000000 +0200
+++ Tk-804.027/TextList/textlist_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5;
 use strict;
diff -Nru Tk-804.027.vanilla/TixGrid/tgdemo_config Tk-804.027/TixGrid/tgdemo_config
--- Tk-804.027.vanilla/TixGrid/tgdemo_config	2003-07-19 10:39:01.000000000 +0200
+++ Tk-804.027/TixGrid/tgdemo_config	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # tixgrid demo
 #	standalone or sourceable in ptksh
 
diff -Nru Tk-804.027.vanilla/WinPhoto/try Tk-804.027/WinPhoto/try
--- Tk-804.027.vanilla/WinPhoto/try	2003-08-24 14:07:26.000000000 +0200
+++ Tk-804.027/WinPhoto/try	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::WinPhoto;
 use Cwd;
diff -Nru Tk-804.027.vanilla/Xlib/drops Tk-804.027/Xlib/drops
--- Tk-804.027.vanilla/Xlib/drops	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/Xlib/drops	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 
 use Tk;
diff -Nru Tk-804.027.vanilla/Xlib/load Tk-804.027/Xlib/load
--- Tk-804.027.vanilla/Xlib/load	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/Xlib/load	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Xlib;
diff -Nru Tk-804.027.vanilla/Xlib/motif_site Tk-804.027/Xlib/motif_site
--- Tk-804.027.vanilla/Xlib/motif_site	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/Xlib/motif_site	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Xlib;
 use Data::Dumper;
diff -Nru Tk-804.027.vanilla/Xlib/tree_demo Tk-804.027/Xlib/tree_demo
--- Tk-804.027.vanilla/Xlib/tree_demo	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/Xlib/tree_demo	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Xlib;
 use Tk::HList;
diff -Nru Tk-804.027.vanilla/add_version Tk-804.027/add_version
--- Tk-804.027.vanilla/add_version	2003-12-14 20:36:35.000000000 +0100
+++ Tk-804.027/add_version	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use File::Find;
 use Getopt::Std;
diff -Nru Tk-804.027.vanilla/adj_demo Tk-804.027/adj_demo
--- Tk-804.027.vanilla/adj_demo	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/adj_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  -w
+#!/usr/bin/perl  -w
 use Tk;
 require Tk::TextUndo;
 
diff -Nru Tk-804.027.vanilla/after_demo Tk-804.027/after_demo
--- Tk-804.027.vanilla/after_demo	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/after_demo	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use strict;
diff -Nru Tk-804.027.vanilla/after_leak Tk-804.027/after_leak
--- Tk-804.027.vanilla/after_leak	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/after_leak	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Devel::Leak;
diff -Nru Tk-804.027.vanilla/animate Tk-804.027/animate
--- Tk-804.027.vanilla/animate	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/animate	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
diff -Nru Tk-804.027.vanilla/autoraise Tk-804.027/autoraise
--- Tk-804.027.vanilla/autoraise	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/autoraise	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/basic_demo Tk-804.027/basic_demo
--- Tk-804.027.vanilla/basic_demo	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/basic_demo	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # use Memory (2440);
 
diff -Nru Tk-804.027.vanilla/bin/defs Tk-804.027/bin/defs
--- Tk-804.027.vanilla/bin/defs	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bin/defs	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 %def   = ();
 %need  = ();
diff -Nru Tk-804.027.vanilla/bin/defunc Tk-804.027/bin/defunc
--- Tk-804.027.vanilla/bin/defunc	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bin/defunc	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 while (<>)
  {
diff -Nru Tk-804.027.vanilla/bin/newstyle Tk-804.027/bin/newstyle
--- Tk-804.027.vanilla/bin/newstyle	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bin/newstyle	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 @files = @ARGV;
 
diff -Nru Tk-804.027.vanilla/bin/perms Tk-804.027/bin/perms
--- Tk-804.027.vanilla/bin/perms	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bin/perms	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use File::Find;
 
diff -Nru Tk-804.027.vanilla/bin/unres Tk-804.027/bin/unres
--- Tk-804.027.vanilla/bin/unres	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bin/unres	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 %done = ();
 %defs = ();
diff -Nru Tk-804.027.vanilla/bin/uselib Tk-804.027/bin/uselib
--- Tk-804.027.vanilla/bin/uselib	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bin/uselib	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use File::Find;
 
diff -Nru Tk-804.027.vanilla/bindtest Tk-804.027/bindtest
--- Tk-804.027.vanilla/bindtest	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bindtest	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Menubar;
 
diff -Nru Tk-804.027.vanilla/build Tk-804.027/build
--- Tk-804.027.vanilla/build	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/build	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 my $make = $Config{'make'};
 my @para;
diff -Nru Tk-804.027.vanilla/bulkedit Tk-804.027/bulkedit
--- Tk-804.027.vanilla/bulkedit	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/bulkedit	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::LabEntry;
diff -Nru Tk-804.027.vanilla/canvas_ps Tk-804.027/canvas_ps
--- Tk-804.027.vanilla/canvas_ps	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/canvas_ps	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/canvas_scroll Tk-804.027/canvas_scroll
--- Tk-804.027.vanilla/canvas_scroll	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/canvas_scroll	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/cbtest Tk-804.027/cbtest
--- Tk-804.027.vanilla/cbtest	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/cbtest	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Tk;
 use Tk::Pretty;
diff -Nru Tk-804.027.vanilla/check_syms Tk-804.027/check_syms
--- Tk-804.027.vanilla/check_syms	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/check_syms	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 use Config;
 my @libs;
diff -Nru Tk-804.027.vanilla/checklen Tk-804.027/checklen
--- Tk-804.027.vanilla/checklen	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/checklen	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use ExtUtils::Manifest "/mani/";
 
diff -Nru Tk-804.027.vanilla/classtree Tk-804.027/classtree
--- Tk-804.027.vanilla/classtree	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/classtree	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::widgets qw(Tree Menubar);
diff -Nru Tk-804.027.vanilla/cleanup Tk-804.027/cleanup
--- Tk-804.027.vanilla/cleanup	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/cleanup	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 find(\&wanted,'.');
 
diff -Nru Tk-804.027.vanilla/clip_bug Tk-804.027/clip_bug
--- Tk-804.027.vanilla/clip_bug	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/clip_bug	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # Show a Tk canvas bug:
 # when scrolling a canvas, clipping for embedded windows
 # uses the embedded windows parent, not the canvas.
diff -Nru Tk-804.027.vanilla/compile/findloader Tk-804.027/compile/findloader
--- Tk-804.027.vanilla/compile/findloader	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/compile/findloader	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 my $sfx = $Config{'lib_ext'};
 foreach my $dir (@INC)
diff -Nru Tk-804.027.vanilla/compile/splitfunc Tk-804.027/compile/splitfunc
--- Tk-804.027.vanilla/compile/splitfunc	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/compile/splitfunc	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use IO::File;
 
 while (<>)
diff -Nru Tk-804.027.vanilla/copydate Tk-804.027/copydate
--- Tk-804.027.vanilla/copydate	2003-07-25 22:16:19.000000000 +0200
+++ Tk-804.027/copydate	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use File::Find;
 use Time::Local;
diff -Nru Tk-804.027.vanilla/cursor_demo Tk-804.027/cursor_demo
--- Tk-804.027.vanilla/cursor_demo	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/cursor_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
diff -Nru Tk-804.027.vanilla/demos/browse Tk-804.027/demos/browse
--- Tk-804.027.vanilla/demos/browse	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/demos/browse	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script generates a directory browser, which lists the working directory and allows you to open files or subdirectories
 # by double-clicking.
diff -Nru Tk-804.027.vanilla/demos/color_editor Tk-804.027/demos/color_editor
--- Tk-804.027.vanilla/demos/color_editor	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/demos/color_editor	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # ColorEditor demonstration.
 
diff -Nru Tk-804.027.vanilla/demos/composite Tk-804.027/demos/composite
--- Tk-804.027.vanilla/demos/composite	2003-07-19 10:39:03.000000000 +0200
+++ Tk-804.027/demos/composite	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use English;
diff -Nru Tk-804.027.vanilla/demos/demos/widtrib/npuz.pl Tk-804.027/demos/demos/widtrib/npuz.pl
--- Tk-804.027.vanilla/demos/demos/widtrib/npuz.pl	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/demos/widtrib/npuz.pl	2005-03-11 20:36:13.000000000 +0100
@@ -5,7 +5,7 @@
 # distribution with permission.  It has been modified slightly to conform
 # to the widget demo standard.
 
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # puz - demonstrate the Grid geometry manager by implementing an n-puzzle.
 #
diff -Nru Tk-804.027.vanilla/demos/demos/widtrib/plop.pl Tk-804.027/demos/demos/widtrib/plop.pl
--- Tk-804.027.vanilla/demos/demos/widtrib/plop.pl	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/demos/widtrib/plop.pl	2005-03-11 20:36:13.000000000 +0100
@@ -5,7 +5,7 @@
 # distribution with permission.  It has been modified slightly to conform
 # to the widget demo standard.
 
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # plot_program - plot a series of continuous functions on a Perl/Tk Canvas.
 #
diff -Nru Tk-804.027.vanilla/demos/dialog Tk-804.027/demos/dialog
--- Tk-804.027.vanilla/demos/dialog	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/dialog	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Demonstrate how to use Dialog to create and manipulate dialog objects.
 # lusol@Lehigh.EDU  95/06/14
diff -Nru Tk-804.027.vanilla/demos/hello Tk-804.027/demos/hello
--- Tk-804.027.vanilla/demos/hello	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/hello	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Simple Tk script to create a button that prints "Hello, world".  Click on
 # the button to terminate the program.
diff -Nru Tk-804.027.vanilla/demos/ixset Tk-804.027/demos/ixset
--- Tk-804.027.vanilla/demos/ixset	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/ixset	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # A nice interface to "xset" to change X server settings
 #
diff -Nru Tk-804.027.vanilla/demos/rmt Tk-804.027/demos/rmt
--- Tk-804.027.vanilla/demos/rmt	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/rmt	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -wT
+#!/usr/bin/perl -wT
 #
 # This script implements a simple remote-control mechanism for Tk
 # applications.  It allows you to select an application and then type
diff -Nru Tk-804.027.vanilla/demos/rolodex Tk-804.027/demos/rolodex
--- Tk-804.027.vanilla/demos/rolodex	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/rolodex	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script was written as an entry in Tom LaStrange's rolodex benchmark.
 # It creates something that has some of the look and feel of a rolodex program,# although it's lifeless and doesn't actually do the rolodex application.
diff -Nru Tk-804.027.vanilla/demos/timer Tk-804.027/demos/timer
--- Tk-804.027.vanilla/demos/timer	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/timer	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script generates a counter with start and stop buttons.  Exit with
 # Ctrl/c or Ctrl/q.
diff -Nru Tk-804.027.vanilla/demos/tom Tk-804.027/demos/tom
--- Tk-804.027.vanilla/demos/tom	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/tom	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Sample program to get Tom Christiansen going with Perl/Tk.
 
diff -Nru Tk-804.027.vanilla/demos/widget Tk-804.027/demos/widget
--- Tk-804.027.vanilla/demos/widget	2004-02-28 17:35:25.000000000 +0100
+++ Tk-804.027/demos/widget	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use 5.008;
 use Config;
diff -Nru Tk-804.027.vanilla/demos/widget_old Tk-804.027/demos/widget_old
--- Tk-804.027.vanilla/demos/widget_old	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/demos/widget_old	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script demonstrates the various widgets provided by Tk, along with many of the features of the Tk toolkit.  This file
 # only contains code to generate the main window for the application, which invokes individual demonstrations.  The code for
diff -Nru Tk-804.027.vanilla/derived_test Tk-804.027/derived_test
--- Tk-804.027.vanilla/derived_test	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/derived_test	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 require Tk::Text;
 
diff -Nru Tk-804.027.vanilla/destroy_test Tk-804.027/destroy_test
--- Tk-804.027.vanilla/destroy_test	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/destroy_test	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use Tk;
diff -Nru Tk-804.027.vanilla/dfpy Tk-804.027/dfpy
--- Tk-804.027.vanilla/dfpy	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/dfpy	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use ExtUtils::Manifest "/mani/";
 use Cwd;
diff -Nru Tk-804.027.vanilla/disbutton Tk-804.027/disbutton
--- Tk-804.027.vanilla/disbutton	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/disbutton	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 package StickyButton;
 require Tk::Widget;
diff -Nru Tk-804.027.vanilla/dnd_demo Tk-804.027/dnd_demo
--- Tk-804.027.vanilla/dnd_demo	2003-07-19 10:39:05.000000000 +0200
+++ Tk-804.027/dnd_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Demo code to exhibit features of dynamic construction and destruction of
 # local (within app.) Drag and Drop sources and sinks. Written for and tested
diff -Nru Tk-804.027.vanilla/dump_test Tk-804.027/dump_test
--- Tk-804.027.vanilla/dump_test	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/dump_test	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 my $top = MainWindow->new;
 my $i   = $top->Photo(-file => Tk->findINC('icon.gif'));
diff -Nru Tk-804.027.vanilla/dupsyms Tk-804.027/dupsyms
--- Tk-804.027.vanilla/dupsyms	2003-07-23 13:42:10.000000000 +0200
+++ Tk-804.027/dupsyms	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 use Cwd;
 my $root = getcwd();
diff -Nru Tk-804.027.vanilla/embed Tk-804.027/embed
--- Tk-804.027.vanilla/embed	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/embed	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use IO::File;
diff -Nru Tk-804.027.vanilla/error_prop Tk-804.027/error_prop
--- Tk-804.027.vanilla/error_prop	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/error_prop	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::ErrorDialog;
diff -Nru Tk-804.027.vanilla/ewidth_demo Tk-804.027/ewidth_demo
--- Tk-804.027.vanilla/ewidth_demo	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/ewidth_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
diff -Nru Tk-804.027.vanilla/fetail Tk-804.027/fetail
--- Tk-804.027.vanilla/fetail	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/fetail	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 my $file = (@ARGV) ? shift : 'README';
 
diff -Nru Tk-804.027.vanilla/findbase Tk-804.027/findbase
--- Tk-804.027.vanilla/findbase	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/findbase	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
diff -Nru Tk-804.027.vanilla/floppy Tk-804.027/floppy
--- Tk-804.027.vanilla/floppy	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/floppy	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use ExtUtils::Manifest "/mani/";
 use Cwd;
diff -Nru Tk-804.027.vanilla/font_demo Tk-804.027/font_demo
--- Tk-804.027.vanilla/font_demo	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/font_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::X11Font;
 use Tk::ROText;
diff -Nru Tk-804.027.vanilla/font_test Tk-804.027/font_test
--- Tk-804.027.vanilla/font_test	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/font_test	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 
diff -Nru Tk-804.027.vanilla/fonts Tk-804.027/fonts
--- Tk-804.027.vanilla/fonts	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/fonts	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Data::Dumper;
diff -Nru Tk-804.027.vanilla/gdb.plx Tk-804.027/gdb.plx
--- Tk-804.027.vanilla/gdb.plx	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/gdb.plx	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 open(INIT,">.gdbinit") || die;
 my $perl = $^X;
diff -Nru Tk-804.027.vanilla/gedi Tk-804.027/gedi
--- Tk-804.027.vanilla/gedi	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/gedi	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 ###############################################################################
 # Copyright (c) 1999  Greg London
diff -Nru Tk-804.027.vanilla/geo_mgr Tk-804.027/geo_mgr
--- Tk-804.027.vanilla/geo_mgr	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/geo_mgr	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/geom Tk-804.027/geom
--- Tk-804.027.vanilla/geom	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/geom	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/grid_adj Tk-804.027/grid_adj
--- Tk-804.027.vanilla/grid_adj	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/grid_adj	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  -w
+#!/usr/bin/perl  -w
 use Tk;
 require Tk::TextUndo;
 require Tk::Adjuster;
diff -Nru Tk-804.027.vanilla/grid_test Tk-804.027/grid_test
--- Tk-804.027.vanilla/grid_test	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/grid_test	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 my $mw = MainWindow->new;
 open(FILE,__FILE__);
diff -Nru Tk-804.027.vanilla/gridbug Tk-804.027/gridbug
--- Tk-804.027.vanilla/gridbug	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/gridbug	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Carp;
 $top = MainWindow->new();
diff -Nru Tk-804.027.vanilla/hackMM Tk-804.027/hackMM
--- Tk-804.027.vanilla/hackMM	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/hackMM	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
diff -Nru Tk-804.027.vanilla/hlfm Tk-804.027/hlfm
--- Tk-804.027.vanilla/hlfm	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/hlfm	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 require Tk::Dirlist;
diff -Nru Tk-804.027.vanilla/hlist_bg Tk-804.027/hlist_bg
--- Tk-804.027.vanilla/hlist_bg	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/hlist_bg	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 require Tk::HList;
 
diff -Nru Tk-804.027.vanilla/iconimg Tk-804.027/iconimg
--- Tk-804.027.vanilla/iconimg	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/iconimg	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/iconwin Tk-804.027/iconwin
--- Tk-804.027.vanilla/iconwin	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/iconwin	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/image_test Tk-804.027/image_test
--- Tk-804.027.vanilla/image_test	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/image_test	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/inline_bitmap Tk-804.027/inline_bitmap
--- Tk-804.027.vanilla/inline_bitmap	2003-07-19 10:39:09.000000000 +0200
+++ Tk-804.027/inline_bitmap	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
diff -Nru Tk-804.027.vanilla/keyWords Tk-804.027/keyWords
--- Tk-804.027.vanilla/keyWords	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/keyWords	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
diff -Nru Tk-804.027.vanilla/labelpack Tk-804.027/labelpack
--- Tk-804.027.vanilla/labelpack	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/labelpack	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::LabEntry;
diff -Nru Tk-804.027.vanilla/lbformat_demo Tk-804.027/lbformat_demo
--- Tk-804.027.vanilla/lbformat_demo	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/lbformat_demo	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Font;
diff -Nru Tk-804.027.vanilla/leak_test Tk-804.027/leak_test
--- Tk-804.027.vanilla/leak_test	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/leak_test	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Devel::Leak;
diff -Nru Tk-804.027.vanilla/lentry Tk-804.027/lentry
--- Tk-804.027.vanilla/lentry	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/lentry	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/linux-progbar-toy Tk-804.027/linux-progbar-toy
--- Tk-804.027.vanilla/linux-progbar-toy	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/linux-progbar-toy	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::widgets qw(ProgressBar);
 
diff -Nru Tk-804.027.vanilla/local_demo Tk-804.027/local_demo
--- Tk-804.027.vanilla/local_demo	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/local_demo	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk qw(MainLoop *widget);
 use Carp;
diff -Nru Tk-804.027.vanilla/ls_table Tk-804.027/ls_table
--- Tk-804.027.vanilla/ls_table	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/ls_table	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Table;
diff -Nru Tk-804.027.vanilla/lsfonts Tk-804.027/lsfonts
--- Tk-804.027.vanilla/lsfonts	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/lsfonts	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Xlib;
diff -Nru Tk-804.027.vanilla/menubar Tk-804.027/menubar
--- Tk-804.027.vanilla/menubar	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/menubar	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/menubug Tk-804.027/menubug
--- Tk-804.027.vanilla/menubug	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/menubug	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
diff -Nru Tk-804.027.vanilla/mitem_test Tk-804.027/mitem_test
--- Tk-804.027.vanilla/mitem_test	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/mitem_test	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.003;
 use English;
diff -Nru Tk-804.027.vanilla/mkExt Tk-804.027/mkExt
--- Tk-804.027.vanilla/mkExt	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/mkExt	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 sub translate
 {
diff -Nru Tk-804.027.vanilla/mkdist Tk-804.027/mkdist
--- Tk-804.027.vanilla/mkdist	2003-12-21 14:04:32.000000000 +0100
+++ Tk-804.027/mkdist	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use ExtUtils::Manifest "/mani/";
diff -Nru Tk-804.027.vanilla/motif_hack Tk-804.027/motif_hack
--- Tk-804.027.vanilla/motif_hack	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/motif_hack	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Data::Dumper;
 use strict;
 use Tk;
diff -Nru Tk-804.027.vanilla/nested_dialog Tk-804.027/nested_dialog
--- Tk-804.027.vanilla/nested_dialog	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/nested_dialog	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Dialog;
diff -Nru Tk-804.027.vanilla/oddmenu_demo Tk-804.027/oddmenu_demo
--- Tk-804.027.vanilla/oddmenu_demo	2003-07-19 10:39:10.000000000 +0200
+++ Tk-804.027/oddmenu_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
diff -Nru Tk-804.027.vanilla/pTk/Tcl-pTk Tk-804.027/pTk/Tcl-pTk
--- Tk-804.027.vanilla/pTk/Tcl-pTk	2003-12-21 11:47:19.000000000 +0100
+++ Tk-804.027/pTk/Tcl-pTk	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use open IO => ':bytes'; # Avoid UTF-8 issues with some perl5.8.0 (RedHat)
 use Carp;
 my $verbose = 0;
diff -Nru Tk-804.027.vanilla/pTk/chext Tk-804.027/pTk/chext
--- Tk-804.027.vanilla/pTk/chext	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/chext	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 open(NEED,"<need") || die;
 while (<NEED>)
diff -Nru Tk-804.027.vanilla/pTk/counts Tk-804.027/pTk/counts
--- Tk-804.027.vanilla/pTk/counts	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/counts	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 sub count
 {
diff -Nru Tk-804.027.vanilla/pTk/deArg Tk-804.027/pTk/deArg
--- Tk-804.027.vanilla/pTk/deArg	2003-07-27 18:42:08.000000000 +0200
+++ Tk-804.027/pTk/deArg	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 $^I='.bak';
 while (<>)
  {
diff -Nru Tk-804.027.vanilla/pTk/findX Tk-804.027/pTk/findX
--- Tk-804.027.vanilla/pTk/findX	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/findX	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { require Exporter };
 
diff -Nru Tk-804.027.vanilla/pTk/findcore Tk-804.027/pTk/findcore
--- Tk-804.027.vanilla/pTk/findcore	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/findcore	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { require Exporter };
 
diff -Nru Tk-804.027.vanilla/pTk/findstrings Tk-804.027/pTk/findstrings
--- Tk-804.027.vanilla/pTk/findstrings	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/findstrings	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 %table = ();
diff -Nru Tk-804.027.vanilla/pTk/from Tk-804.027/pTk/from
--- Tk-804.027.vanilla/pTk/from	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/from	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use File::Find;
diff -Nru Tk-804.027.vanilla/pTk/genexc Tk-804.027/pTk/genexc
--- Tk-804.027.vanilla/pTk/genexc	2003-07-19 10:39:11.000000000 +0200
+++ Tk-804.027/pTk/genexc	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 my %mTk;
 my $dir;
diff -Nru Tk-804.027.vanilla/pTk/mTk/tclmerge Tk-804.027/pTk/mTk/tclmerge
--- Tk-804.027.vanilla/pTk/mTk/tclmerge	2003-07-19 10:39:26.000000000 +0200
+++ Tk-804.027/pTk/mTk/tclmerge	2005-03-11 20:36:25.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
diff -Nru Tk-804.027.vanilla/pTk/mTk/tixmerge Tk-804.027/pTk/mTk/tixmerge
--- Tk-804.027.vanilla/pTk/mTk/tixmerge	2003-07-19 10:39:31.000000000 +0200
+++ Tk-804.027/pTk/mTk/tixmerge	2005-03-11 20:36:25.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
diff -Nru Tk-804.027.vanilla/pTk/makeenglish Tk-804.027/pTk/makeenglish
--- Tk-804.027.vanilla/pTk/makeenglish	2003-07-19 10:39:39.000000000 +0200
+++ Tk-804.027/pTk/makeenglish	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # BEGIN { push(@INC, qw(../../../lib ../../lib ../lib lib)) }
 
 die "Must be executed from Tk/pTk distribution directory\n" unless -f "Lang.h";
diff -Nru Tk-804.027.vanilla/pTk/mkVFunc Tk-804.027/pTk/mkVFunc
--- Tk-804.027.vanilla/pTk/mkVFunc	2004-02-29 12:22:01.000000000 +0100
+++ Tk-804.027/pTk/mkVFunc	2005-03-11 20:36:37.220727416 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 
diff -Nru Tk-804.027.vanilla/pTk/mkXinfo Tk-804.027/pTk/mkXinfo
--- Tk-804.027.vanilla/pTk/mkXinfo	2003-07-19 10:39:39.000000000 +0200
+++ Tk-804.027/pTk/mkXinfo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 $file = shift;
 
diff -Nru Tk-804.027.vanilla/pTk/mkexc Tk-804.027/pTk/mkexc
--- Tk-804.027.vanilla/pTk/mkexc	2003-07-19 10:39:39.000000000 +0200
+++ Tk-804.027/pTk/mkexc	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use vars qw(%ref %def);
 require "process_object";
diff -Nru Tk-804.027.vanilla/pTk/mkmethods Tk-804.027/pTk/mkmethods
--- Tk-804.027.vanilla/pTk/mkmethods	2003-07-19 10:39:39.000000000 +0200
+++ Tk-804.027/pTk/mkmethods	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 $dst = shift;
 
diff -Nru Tk-804.027.vanilla/pTk/p4e Tk-804.027/pTk/p4e
--- Tk-804.027.vanilla/pTk/p4e	2003-07-28 20:48:04.000000000 +0200
+++ Tk-804.027/pTk/p4e	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 use Cwd;
 my $path = getcwd;
diff -Nru Tk-804.027.vanilla/pTk/tdiff Tk-804.027/pTk/tdiff
--- Tk-804.027.vanilla/pTk/tdiff	2003-07-20 18:00:11.000000000 +0200
+++ Tk-804.027/pTk/tdiff	2005-03-11 20:36:37.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 sub fail
 {
diff -Nru Tk-804.027.vanilla/pTk/why Tk-804.027/pTk/why
--- Tk-804.027.vanilla/pTk/why	2003-07-19 10:39:40.000000000 +0200
+++ Tk-804.027/pTk/why	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
diff -Nru Tk-804.027.vanilla/pfm Tk-804.027/pfm
--- Tk-804.027.vanilla/pfm	2003-07-19 10:39:40.000000000 +0200
+++ Tk-804.027/pfm	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use POSIX qw(getcwd);
 
 use Tk;
diff -Nru Tk-804.027.vanilla/pipe_test Tk-804.027/pipe_test
--- Tk-804.027.vanilla/pipe_test	2003-07-19 10:39:40.000000000 +0200
+++ Tk-804.027/pipe_test	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 require Tk::IO;
diff -Nru Tk-804.027.vanilla/pixmap_test Tk-804.027/pixmap_test
--- Tk-804.027.vanilla/pixmap_test	2003-07-19 10:39:40.000000000 +0200
+++ Tk-804.027/pixmap_test	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 require Tk::Pixmap;
diff -Nru Tk-804.027.vanilla/pod/Text.pod Tk-804.027/pod/Text.pod
--- Tk-804.027.vanilla/pod/Text.pod	2004-03-10 09:38:08.000000000 +0100
+++ Tk-804.027/pod/Text.pod	2005-03-11 20:36:16.000000000 +0100
@@ -2151,7 +2151,7 @@
 
 For example:
 
- #!/usr/local/bin/perl -w
+ #!/usr/bin/perl -w
  use POSIX 'acos';
  use Tk;
  use strict;
diff -Nru Tk-804.027.vanilla/pod/extract/extpod Tk-804.027/pod/extract/extpod
--- Tk-804.027.vanilla/pod/extract/extpod	2003-07-19 10:39:41.000000000 +0200
+++ Tk-804.027/pod/extract/extpod	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Getopt::Std;
 use ExtUtils::Manifest "/mani/";
diff -Nru Tk-804.027.vanilla/pod/recat Tk-804.027/pod/recat
--- Tk-804.027.vanilla/pod/recat	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/pod/recat	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 my $sec;
 my %cat;
diff -Nru Tk-804.027.vanilla/pod/subs Tk-804.027/pod/subs
--- Tk-804.027.vanilla/pod/subs	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/pod/subs	2005-03-11 20:36:15.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use blib;
 use Tk;
 use strict;
diff -Nru Tk-804.027.vanilla/point_test Tk-804.027/point_test
--- Tk-804.027.vanilla/point_test	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/point_test	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
diff -Nru Tk-804.027.vanilla/popup Tk-804.027/popup
--- Tk-804.027.vanilla/popup	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/popup	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use Tk;
diff -Nru Tk-804.027.vanilla/popup_menu Tk-804.027/popup_menu
--- Tk-804.027.vanilla/popup_menu	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/popup_menu	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use Tk;
diff -Nru Tk-804.027.vanilla/prop_demo Tk-804.027/prop_demo
--- Tk-804.027.vanilla/prop_demo	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/prop_demo	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/ptked Tk-804.027/ptked
--- Tk-804.027.vanilla/ptked	2004-03-17 22:30:03.000000000 +0100
+++ Tk-804.027/ptked	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Socket;
 use IO::Socket;
diff -Nru Tk-804.027.vanilla/ptknews Tk-804.027/ptknews
--- Tk-804.027.vanilla/ptknews	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/ptknews	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use Tk;
diff -Nru Tk-804.027.vanilla/scale_test Tk-804.027/scale_test
--- Tk-804.027.vanilla/scale_test	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/scale_test	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/scrolled_demo Tk-804.027/scrolled_demo
--- Tk-804.027.vanilla/scrolled_demo	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/scrolled_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use strict;
diff -Nru Tk-804.027.vanilla/sillymenu Tk-804.027/sillymenu
--- Tk-804.027.vanilla/sillymenu	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/sillymenu	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Data::Dumper;
diff -Nru Tk-804.027.vanilla/sockets Tk-804.027/sockets
--- Tk-804.027.vanilla/sockets	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/sockets	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use IO::Socket;
 use Tk::IO;
diff -Nru Tk-804.027.vanilla/spinbox Tk-804.027/spinbox
--- Tk-804.027.vanilla/spinbox	2003-07-23 19:11:51.000000000 +0200
+++ Tk-804.027/spinbox	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::widgets qw(Button Spinbox);
diff -Nru Tk-804.027.vanilla/stripblib Tk-804.027/stripblib
--- Tk-804.027.vanilla/stripblib	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/stripblib	2005-03-11 20:36:13.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 find(\&strip,'blib/arch');
 
diff -Nru Tk-804.027.vanilla/style_test Tk-804.027/style_test
--- Tk-804.027.vanilla/style_test	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/style_test	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::HList;
diff -Nru Tk-804.027.vanilla/submethods Tk-804.027/submethods
--- Tk-804.027.vanilla/submethods	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/submethods	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 $meth = "pTk/Methods.def";
 
 %sub = ();
diff -Nru Tk-804.027.vanilla/symbols Tk-804.027/symbols
--- Tk-804.027.vanilla/symbols	2003-07-19 10:39:42.000000000 +0200
+++ Tk-804.027/symbols	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Config;
 
diff -Nru Tk-804.027.vanilla/t/fileevent.t Tk-804.027/t/fileevent.t
--- Tk-804.027.vanilla/t/fileevent.t	2004-03-27 11:21:15.000000000 +0100
+++ Tk-804.027/t/fileevent.t	2005-03-11 20:36:24.000000000 +0100
@@ -1,7 +1,7 @@
 BEGIN { $^W = 1; $| = 1;}
 use Test;
 
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Severely hacked test based on IPC example from
 # Advanced Perl/Tk Programming.
@@ -294,7 +294,7 @@
 
 } # end start_ipc_helper
 __DATA__
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use 5.005;
 #use IPADM;
diff -Nru Tk-804.027.vanilla/table_demo Tk-804.027/table_demo
--- Tk-804.027.vanilla/table_demo	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/table_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Table;
diff -Nru Tk-804.027.vanilla/tainted Tk-804.027/tainted
--- Tk-804.027.vanilla/tainted	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tainted	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -T
+#!/usr/bin/perl -T
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/tcl2perl Tk-804.027/tcl2perl
--- Tk-804.027.vanilla/tcl2perl	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tcl2perl	2005-03-11 20:36:16.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Highly experimental tcl -> perl converter, aimed at eventually
 # completeing Tix merge
diff -Nru Tk-804.027.vanilla/tcolour Tk-804.027/tcolour
--- Tk-804.027.vanilla/tcolour	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tcolour	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/tent Tk-804.027/tent
--- Tk-804.027.vanilla/tent	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tent	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::widgets qw(Button);
diff -Nru Tk-804.027.vanilla/text_demo Tk-804.027/text_demo
--- Tk-804.027.vanilla/text_demo	2003-09-28 11:48:05.000000000 +0200
+++ Tk-804.027/text_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::Text;
diff -Nru Tk-804.027.vanilla/tie_basic Tk-804.027/tie_basic
--- Tk-804.027.vanilla/tie_basic	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tie_basic	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # use Memory (2440);
 
diff -Nru Tk-804.027.vanilla/tiehandle Tk-804.027/tiehandle
--- Tk-804.027.vanilla/tiehandle	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tiehandle	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
diff -Nru Tk-804.027.vanilla/tietext Tk-804.027/tietext
--- Tk-804.027.vanilla/tietext	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/tietext	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 {package Tk::Text::Contents;
diff -Nru Tk-804.027.vanilla/time Tk-804.027/time
--- Tk-804.027.vanilla/time	2003-07-19 10:39:43.000000000 +0200
+++ Tk-804.027/time	2005-03-11 20:36:20.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk::Stats 'Start';
 use strict;
 use Tk;
diff -Nru Tk-804.027.vanilla/tiny Tk-804.027/tiny
--- Tk-804.027.vanilla/tiny	2003-09-08 21:21:53.000000000 +0200
+++ Tk-804.027/tiny	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::widgets qw(Button);
diff -Nru Tk-804.027.vanilla/tixwm_demo Tk-804.027/tixwm_demo
--- Tk-804.027.vanilla/tixwm_demo	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/tixwm_demo	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Dialog;
diff -Nru Tk-804.027.vanilla/tkfontsel Tk-804.027/tkfontsel
--- Tk-804.027.vanilla/tkfontsel	2004-03-24 15:29:32.000000000 +0100
+++ Tk-804.027/tkfontsel	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Font;
diff -Nru Tk-804.027.vanilla/tkperldoc Tk-804.027/tkperldoc
--- Tk-804.027.vanilla/tkperldoc	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/tkperldoc	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use Tk;
diff -Nru Tk-804.027.vanilla/tkpsh Tk-804.027/tkpsh
--- Tk-804.027.vanilla/tkpsh	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/tkpsh	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.001;
 use IO::Handle;
diff -Nru Tk-804.027.vanilla/tktail Tk-804.027/tktail
--- Tk-804.027.vanilla/tktail	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/tktail	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use IO::File;
diff -Nru Tk-804.027.vanilla/two_main Tk-804.027/two_main
--- Tk-804.027.vanilla/two_main	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/two_main	2005-03-11 20:36:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Xrm;
diff -Nru Tk-804.027.vanilla/uninstall Tk-804.027/uninstall
--- Tk-804.027.vanilla/uninstall	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/uninstall	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 BEGIN {exec('perl5.002',$0,@ARGV)  unless ($] >= 5.002)}
 use Config;
 use Getopt::Long;
diff -Nru Tk-804.027.vanilla/ventry_demo Tk-804.027/ventry_demo
--- Tk-804.027.vanilla/ventry_demo	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/ventry_demo	2005-03-11 20:36:17.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
diff -Nru Tk-804.027.vanilla/xlib_demo Tk-804.027/xlib_demo
--- Tk-804.027.vanilla/xlib_demo	2003-07-19 10:39:44.000000000 +0200
+++ Tk-804.027/xlib_demo	2005-03-11 20:36:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
